### DISCUSSION TITLE
freerdp3 settings for rdp (WINDOWS11 with vGPU )

### DISCUSSION THREAD

**Discussion by kramervanessa** [FreeRDP/FreeRDP#10157] [Q&A]
How to get the best output on the thinclient (pi5), ubuntu 24.04, kernel 6.8 to connect to my virtual desktop.

xfreerdp3 /u:user /v:IP /gdi:hw +gfx /gfx:AVC444:on /compression-level:2 +video /f

With this settings its not perfect, videos are not running smoothly , Any setting to make it better?

Settings in the group policy:
![Bildschirmfoto vom 2024-05-02 12-45-58](https://github.com/FreeRDP/FreeRDP/assets/126567170/23388053-25d5-48bc-b512-c828bb9df641)

**Comment by akallabeth** [comment]
don´t use any argument at all.
1. just use `xfreerdp3 /v:hostname /u:user /f` (with luck `/gdi:hw` does something for you, but only with luck)
2. depending on which mode works better enable or disable `AVC444` in `GPO` on your windows (you then get `AVC444` with better compression/lower bandwidth usage or `RFX-progressive` with high bandwidth usage but lower `CPU` requirements)
3. done

from personal experience, run `xfreerdp` with `xwayland` then it usually has better screen update timing (at least on my desktop machines) or you can try `sdl-freerdp` with native SDL/wayland (if you don´t need clipboard that is, that is a thing still lacking there)
